src/firefly/tests/test_xafs_scan.py

```diff
@@ -234,12 +234,12 @@ def check_item(item):
             # Check whether time is calculated correctly for a single scan
             assert int(float(display.ui.label_hour_scan.text())) == 0
```

Comment:
A better way would be to test that the text equals the string ``0``. E.g.:

```python
assert display.ui.label_hour_scan.text() == "0"
```

Otherwsie, let's say the code is broken and ``label.text()`` produces "0.85". First it gets converted to the float 0.85, then int(0.85) is ``0`` so the test still passes.

Reply:
fixed